eos/nonlocal-form-factors/Makefile.am

```diff
@@ -0,0 +1,74 @@
+CLEANFILES = *~
+MAINTAINERCLEANFILES = Makefile.in
+
+AM_CXXFLAGS = @AM_CXXFLAGS@
+AM_LDFLAGS = @AM_LDFLAGS@
+
+lib_LTLIBRARIES = libeosnonlocalformfactors.la
+libeosnonlocalformfactors_la_SOURCES = \
+	charm-loops.cc charm-loops.hh \
+	charm-loops-analytic.cc charm-loops-impl.hh \
+	charm-loops-analytic-F279b-helper.cc \
+	charm-loops-analytic-F27b.cc \
+	charm-loops-analytic-F29b.cc \
+	charm-loops-analytic-F27d-helper.cc \
+	charm-loops-analytic-F29d-helper.cc \
+	charm-loops-analytic-weight4.cc \
+	charm-loops-analytic-weight4wx3wx4.cc \
+	charm-loops-analytic-weight4w4w5w7.cc \
+	charm-loops-analytic-F279d-helper.cc \
+	charm-loops-analytic-F27d.cc \
+	charm-loops-analytic-F29d.cc \
+	hard-scattering.cc hard-scattering.hh \
+	long-distance.cc long-distance.hh \
+	observables.cc observables.hh \
+	nonlocal-formfactors.hh nonlocal-formfactors.cc nonlocal-formfactors-fwd.hh \
+	nonlocal-formfactors-p-to-p.cc \
+	nonlocal-formfactors-p-to-v.cc
+libeosnonlocalformfactors_la_CXXFLAGS = $(AM_CXXFLAGS) $(GSL_CXXFLAGS)
+libeosnonlocalformfactors_la_LDFLAGS = $(AM_LDFLAGS) $(GSL_LDFLAGS)
+libeosnonlocalformfactors_la_LIBADD = \
+	$(top_builddir)/eos/utils/libeosutils.la \
+	$(top_builddir)/eos/maths/libeosmaths.la \
+	$(top_builddir)/eos/models/libeosmodels.la \
+	$(top_builddir)/eos/form-factors/libeosformfactors.la \
+	-lboost_filesystem -lboost_system \
+	-lgsl -lgslcblas -lm
+
+include_eos_nonlocalformfactorsdir = $(includedir)/eos/rare-b-decays
```

Comment:
Typo, should be installed to ``$(includedir)/eos/nonlocal-form-factors``.

Reply:
Yes! Thanks :)